interactive/views.py

```diff
@@ -225,6 +225,18 @@ def dispatch(self, request, *args, **kwargs):
         if not self.analysis_request.report:
             raise Http404
 
+        # we can only edit reports if they don't have any publish requests or
+        # the latest one is rejected
+        latest_publish_request = self.analysis_request.report.publish_requests.order_by(
+            "-created_at"
+        ).first()
+        if latest_publish_request and (
+            latest_publish_request.decision is None
```

Comment:
What is the semantics of `decision` being `None`? Is it an outstanding request? It might be helpful to make all the cases clear in the comment.

Reply:
I've got a nicer API for this (it'll be `latest_publish_request.is_pending`) coming in a subsequent PR since it's popped up a few times now!  I'll update that PR to cover this case 👍